modules/yahoosspBidAdapter.js

```diff
@@ -244,7 +275,9 @@ function generateOpenRtbObject(bidderRequest, bid) {
       regs: {
         ext: {
           'us_privacy': bidderRequest.uspConsent ? bidderRequest.uspConsent : '',
-          gdpr: bidderRequest.gdprConsent && bidderRequest.gdprConsent.gdprApplies ? 1 : 0
+          gdpr: bidderRequest.gdprConsent && bidderRequest.gdprConsent.gdprApplies ? 1 : 0,
+          gpp: bidderRequest.gppConsent.gppString,
+          gpp_sid: bidderRequest.gppConsent.applicableSections
```

Comment:
@slimkrazy 

Hi, I work for Magnite and during version testing between 7.50.0 and 7.54.0 we noticed many of our pubs had `yahoossp` bids stop filling.

This is the only PR for the bid adapter during that time and I think it might point to this spot here


The object `gppConsent` is not guaranteed to be defined.

For example on one of our publisher sites we see it throws the following error:

![image](https://github.com/prebid/Prebid.js/assets/13972794/6266fe50-2daf-4165-8089-8c1c63c47a86)

An easy fix for this is to change this to be like so:

```js
          gpp: bidderRequest.gppConsent?.gppString,
          gpp_sid: bidderRequest.gppConsent?.applicableSections
```

Let me know if this makes sense.

I tested it using Chrome Overrides and can see bid requests going out now:

![image](https://github.com/prebid/Prebid.js/assets/13972794/95ff9ffd-9caf-4832-ba1a-c9ce54bd976f)